### ISSUE TITLE
Do you have the Linux/GCC   port files for Prof. Zimmermann's work

### ISSUE THREAD

**Issue by gthaker** [janwillemCA/UCOS-II-x86-#2]
I can not find his latest (v3.4) Linux/GCC port files anywhere. (ANd he is not longer making ANY downloads avail. on his website.)

MinGW being saved here is nice, but wish I can find LINUX/GCC port of his.

**Comment by janwillemCA** [comment]
No, I don't. I Was working with in when I used a Windows environment. Never took the effort to try it on Linux with GCC
